src/lib/create/date-from-array.js

```diff
@@ -1,21 +1,36 @@
 export function createDate (y, m, d, h, M, s, ms) {
+    var date;
+    // the date constructor remaps years 0-99 to 1900-1999
+    var remapYears = (y < 100 && y >= 0);
     // can't just apply() to create a date:
     // https://stackoverflow.com/q/181348
-    var date = new Date(y, m, d, h, M, s, ms);
+    if (arguments.length === 3) {
+        // Special-case: handles dates that don't start at midnight
+        date = new Date(remapYears ? y + 400 : y, m, d);
```

Comment:
So this behaves differently depending on whether those lower-order units are undefined?

Reply:
I thought it did, but it doesn't.  I've removed it now.